sss/puma.py

```diff
@@ -0,0 +1,232 @@
+import os
+import glob
+
+import pandas as pd
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Float,
+    Boolean)
+
+import warnings
+from pandas.core.common import SettingWithCopyWarning
+warnings.simplefilter(action="ignore", category=SettingWithCopyWarning)
+from .base import Base, AutomappedDB, DeclarativeDB
+from .base import db_url as default_db_url
+#from sss import PUMA
+
+# declare PUMA data columns and data type
+class PUMA(Base):
+    '''
+    Attributes
+    ----------
+    sl: str
+        Summary Level Code
```

Comment:
I don't know what this means. We need a more descriptive docstring and maybe a more descriptive column name.